hack/istio-install-openshift.sh

```diff
@@ -54,8 +55,6 @@ oc expose svc istio-ingress
 oc create -f install/kubernetes/addons/prometheus.yaml
 oc create -f install/kubernetes/addons/grafana.yaml
 oc create -f install/kubernetes/addons/servicegraph.yaml
-## Workaround for servicegraph bug https://github.com/istio/issues/issues/179
```

Comment:
@gbaufake after pulling your change, I'm getting the servicegraph error documented in the github issue. Is there a reason you deleted this workaround? AFAICS, I think we still need it (at least it looks like I still need it)

Reply:
might be my problem - I had both instio installations so I think it was picking up 0.5.0

Reply:
never mind - problem was I was using 0.5.0. 0.5.1 doesn't need the workaround.